src/explainer_prep.R

```diff
@@ -0,0 +1,193 @@
+                                       width, height, font_legend, text_color, blue_label, orange_label){
+
+  plot_margin <- 0.025
+  # background
+  canvas <- grid::rectGrob(
+    x = 0, y = 0, 
+    width = 16, height = 9,
+    gp = grid::gpar(fill = color_bknd, alpha = 1, col = color_bknd)
+  )
+  
+  #arrows
+  (normal_range_arrow <- ggplot() + 
+      theme_void()+
+      # add arrow using `geom_curve()`
+      geom_curve(aes(x = 13, y = 3,
+                     xend = 11, yend = 3),
+                 arrow = grid::arrow(length = unit(0.2, 'lines')), 
+                 curvature = 0, angle = 100, ncp = 10,
+                 color = text_color, linewidth = 0.2))
+  
+  (low_range_arrow <- ggplot() + 
+      theme_void()+
+      # add arrow using `geom_curve()`
+      geom_curve(aes(x = 13, y = 3,
+                     xend = 11, yend = 3),
+                 arrow = grid::arrow(length = unit(0.2, 'lines')), 
+                 curvature = 0, angle = 100, ncp = 10,
+                 color = "#A84E0B", linewidth = 0.2))
+  
+  (high_range_arrow <- ggplot() + 
+      theme_void()+
+      # add arrow using `geom_curve()`
+      geom_curve(aes(x = 13, y = 3,
+                     xend = 11, yend = 3),
+                 arrow = grid::arrow(length = unit(0.2, 'lines')), 
+                 curvature = 0, angle = 100, ncp = 10,
+                 color = "#002D5E", linewidth = 0.2))
+  
+  og_plot_png <- magick::image_read(national_plot_png)
+  # compose final plot
+  ggdraw(ylim = c(0,1), 
+         xlim = c(0,1)) +
+    # a white background
+    draw_grob(canvas,
+              x = 0, y = 1,
+              height = 1, width = 1,
+              hjust = 0, vjust = 1) +
+    draw_image(og_plot_png, 
+               x = 0, y = 0, 
+               width = 1, 
+               hjust = 0, vjust = 0, 
+               halign = 0, valign = 0)+
+    draw_label(explainer_label, 
+               x = 0.5, y = 0.96, 
+               size = 5.5, 
+               hjust = 0, vjust = 1,
+               fontfamily = font_legend,
+               color = "#000000")+ #text_color) +
+    draw_label(blue_label, 
+               x = 0.5, y = 0.91, 
+               size = 5.5, 
+               hjust = 0, vjust = 1,
+               fontfamily = font_legend,
+               color = "#002D5E")+
+    draw_label(orange_label, 
+               x = 0.5, y = 0.88, 
+               size = 5.5, 
+               hjust = 0, vjust = 1,
+               fontfamily = font_legend,
+               color = "#A84E0B")+
+    draw_label("Low\nStreamflow",
+               x = 0.84, y = 0.355, 
+               size = 5.5, 
+               hjust = 0.5, vjust = 1,
+               fontfamily = font_legend,
+               color = "#A84E0B") +
+    draw_label("High\nStreamflow",
+               x = 0.84, y = 0.76, 
+               size = 5.5, 
+               hjust = 0.5, vjust = 1,
+               fontfamily = font_legend,
+               color = "#002D5E") +
+    draw_label("Typical\nStreamflow",
```

Comment:
Thinking about the placement of this label and if it y should be more dynamically placed. We may have very wet or very dry months where that label will need to be moved up or down.  For example, Oct2022 was really dry so that "typical label" would need to moved higher on the y-axis.
![oct22_flowtiles](https://github.com/user-attachments/assets/75c707e2-fb4e-4b86-b4f0-03e8f593d967)

to alleviate this, lets add a fxn parameter  like `typical_label_ypos` or something like that so we can edit that quickly in `cowplot_national_explainer()`